### PR TITLE
[ET-1542] Enhancement - TicketsCommerce Stripe Dashboard now shows additional meta data.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
### 🎫 Ticket

[ET-1542] 

### 🗒️ Description

* Added some extra fields in meta data for Stripe.
* Now it shows `purhcaser_name`, `purhchaser_email`, `event_name`, `event_url` and `ticket_names` in meta data section in Stripe dashboard.

### 🎥 Artifacts 
 📷 screenshot(s):

Screen Shot 2022-07-28 at 3 33 46 AM

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1542]: https://theeventscalendar.atlassian.net/browse/ET-1542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php [hunk 1]
```diff
@@ -215,4 +210,46 @@ public function delete_payment_intent_transient() {
...
+	 *
+	 * @since TBD
+	 *
+	 * @param \WP_Post $order The Order data.
+	 * @param array $payment_intent The Payment intent.
+	 *
+	 * @return array
+	 */
+	protected function get_updated_metadata( \WP_Post $order, array $payment_intent ) {
+		// Add the Order ID as metadata to the Payment Intent
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php:225]:
```suggestion
		// Add the Order ID as metadata to the Payment Intent.
```

File: src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php [hunk 2]
```diff
@@ -215,4 +210,46 @@ public function delete_payment_intent_transient() {
...
+
+	/**
+	 * Get the additional metadata for the payment intent.
+	 *
+	 * @since TBD
+	 *
+	 * @param \WP_Post $order The Order data.
+	 * @param array $payment_intent The Payment intent.
+	 *
+	 * @return array
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php:222]: Maybe we can document a bit the format of the array, or the keys that it should contain?

**Review by juanfra — APPROVED** [review]
Looking really good!

File: src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php (hunk 2 above)
Comment (rafsuntaskin) [reply src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php:222]: We don't have any required set of metadata, anything with a key-value pair should be valid as long as the hooked filter is appending data to these values.

**Review by codingmusician — APPROVED** [review]

**Pull request merged**